test/unit/PanoViewer/PanoViewer.spec.js

```diff
@@ -81,7 +83,25 @@ describe("PanoViewer", function() {
 
 		IT("should config cubemap layout", done => {
 			// Given
-			panoViewer = new PanoViewer(target, {
+			var MockedPanoViewer = PanoViewerInjector(
```

Comment:
Although test code is not target of `lint`, I want you to follow coding convention.

Reply:
![image](https://user-images.githubusercontent.com/3255471/35761723-233feaf0-08cf-11e8-80b4-2f43125dcdc2.png)

Do not need to fix all code, It's better to fix newly made code.